cmd/caib/main.go

```
@@ -775,6 +780,17 @@ func downloadArtifacts(imageBuild *automotivev1.ImageBuild) {
 		return
 	}
 
+	// If ExposeArtifacts is true, print the route URL
+	if artifactsRoute {
+		fmt.Println("this is the status: %s", imageBuild.Status)
```

golangci-lint (1.64.8)

printf: fmt.Println call has possible Printf formatting directive %s

(govet)